### PR TITLE
n+g fixes

### PR DESCRIPTION
**Pull request by ragusa**
some issues remain:
1) the --plot argument needs to be fixed
2) several examples do not run because the repo does not contain the necessary files

### CODE REVIEW

**Comment by Naktakala** [comment]
Did you run example1b.sh after these changes? Does it throw a python error for you?

File: njoy_processor/Utils_ReadNJOYOutput.py [hunk 1]
```diff
@@ -360,38 +368,68 @@ def ReadNJOYfile(njoy_filename = "output", verbose = False):
                 cross_sections["delayed_chi"] = \
                     ProcessDelayedChi(nL, file_lines)
 
+###---------matrix ------------------------------------------------------------
+            # caveat: the pp values from transfer(mf26) = 2x the pp from the xsec(mf23)
+            # caveat: the n,2n values from transfer(mf8/mt16) = 2x the n,2n from the xsec(mf3/mt16)
             if words[num_words - 1] == "matrix":
-                transfer_matrices[words[num_words - 3]] = \
+                particule_type = words[num_words - 2]
```
Comment (Naktakala) [inline njoy_processor/Utils_ReadNJOYOutput.py]: `particule_type` is this `particle_type` with a french accent?

File: njoy_processor/Utils_ReadNJOYOutput.py [hunk 2]
```diff
@@ -360,38 +368,68 @@ def ReadNJOYfile(njoy_filename = "output", verbose = False):
...
+                reaction_type  = words[num_words - 3]
+                if particule_type=='free-gas':
+                    particule_type = 'neutron'
+                    reaction_type  = 'free-gas'
+                transfer_matrices[particule_type][reaction_type] = \
                     ProcessTransferMatrix(nL, file_lines)
```
Comment (Naktakala) [inline njoy_processor/Utils_ReadNJOYOutput.py:388]: This bugs with the S(alpha,beta) examples.

**Review by Naktakala — CHANGES_REQUESTED** [review]

File: njoy_processor/Utils_ReadNJOYOutput.py (hunk 2 above)
Comment (ragusa) [reply njoy_processor/Utils_ReadNJOYOutput.py:388]: The original code produces in the console

image

However, the code has those lines

image

The example does not use that s_ab keyword in the cross section dict, only in the transfer dict, so I will leave it as is as it causes for issues (for lacking of testing)

File: njoy_processor/Utils_ReadNJOYOutput.py (hunk 1 above)
Comment (ragusa) [reply njoy_processor/Utils_ReadNJOYOutput.py]: yes. I can change that.

**Review by Naktakala — APPROVED** [review]

**Pull request merged**